components/card/collapsible/src/index.scss

```diff
@@ -45,25 +46,70 @@
     }
 
     &.is-expanded {
+      display: flex;
       flex-direction: row;
 
       .sui-CardCollapsible-header {
         &Image {
-          margin-right: $m-s;
+          height: 80px;
+          margin: auto $m-m;
+        }
+
+        &Info {
+          overflow: hidden;
+          padding-right: $p-l;
+          width: 100%;
         }
 
         &InfoItem {
           font-size: $fz-card-collapsible-info-item-expanded;
-          margin-bottom: $m-s;
+          line-height: $lh-card-collapsible-info-item-expanded;
+          margin-bottom: 0;
+          word-break: break-all;
 
           &:last-child {
             margin: $m-m 0 0;
           }
         }
       }
+
+      .sui-CardCollapsible-infoItemIcon {
+        height: $sz-card-collapsible-info-item-icon-expanded;
+        margin-right: $m-s;
+        vertical-align: baseline;
+        width: $sz-card-collapsible-info-item-icon-expanded;
+      }
     }
   }
 
+  &.is-expanded {
+    &.sui-CardCollapsible--boxShadow {
+      box-shadow: $bxsh-card-collapsible-expanded;
+    }
+  }
+
+  &-expandButton {
```

Comment:
Don't override internal properties by css. Use one the the provided sizes

Reply:
OK, I am removing this class. It was just because in our designs the font-size of the card'2 button is bigger than the defined in atom button by 2px, but I think we can survive without it :D

Reply:
@betobe The spacing is correct (16px):
```
&Close {
      display: flex;
      justify-content: flex-end;
      padding: $p-l;
    }
```
It just seems bigger because a copied a sample in the playground which had an extra padding around the collapsible card: `<div style={{backgroundColor:'white', padding: 16}}>`.